docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,76 @@
+| kyma-source-environment | This is the business event's source environment which has been delivered to the micro-service. e.g. `ec-qa` |
+| kyma-source-namespace | This is the business event's source namespace which has been delivered to the micro-service. e.g. `local.kyma.commerce` |
+| kyma-source-type | This is the business event's source type which has been delivered to the micro-service. e.g. `commerce` |
+| kyma-subscription | This is the subscription name which is defined in the subscription contract/CRD and this business event has been published to its subscribers. e.g. `example-subscription` |
+| kyma-topic | This is the topic name used internally by the kyma event bus to deliver the messages to the subscribers via a message bus. e.g. `ec-qa.local\.kyma\.commerce.commerce.test-event-bus.v1` |
+| x-b3-flags | This header is used by the Zipkin tracer in Envoy. The encode one or more options. For example, Debug is encoded as X-B3-Flags: 1. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `0` |
+| x-b3-parentspanid | This header is used by the Zipkin tracer in Envoy. The ParentSpanId is 64-bit in length and indicates the position of the parent operation in the trace tree. When the span is the root of the trace tree, the ParentSpanId is absent. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation) e.g. `3f52e7d591bf74e8` |
+| x-b3-sampled | This header is used by the Zipkin tracer in Envoy. When the Sampled flag is either not specified or set to 1, the span will be reported to the tracing system. Once Sampled is set to 0 or 1, the same value should be consistently sent downstream. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `1` |
+| x-b3-spanid | This header is used by the Zipkin tracer in Envoy. The SpanId is 64-bit in length and indicates the position of the current operation in the trace tree. The value should not be interpreted: it may or may not be derived from the value of the TraceId. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `7350c92dc6fa825e` |
+|x-b3-traceid | This header is used by the Zipkin tracer in Envoy. The TraceId is 64-bit in length and indicates the overall ID of the trace. Every span in a trace shares this ID. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `02c04beed9aa6c53` |
+| x-request-id | Randomly generated an ID for identifies the HTTP request delivering the business event. e.g. `ec0ffa2a-13b6-9e63-9563-12e5c0e27e38` |
+| x-envoy-decorator-operation | If this header is present on ingress requests, its value will override any locally defined operation (span) name on the server span generated by the tracing mechanism. Similarly, if this header is present on an egress response, its value will override any locally defined operation (span) name on the client span. e.g. `default-route` |
```

Comment:
``If this header is present in Ingress requests, its value overrides any locally defined operation (span) name on the server span generated by the tracing mechanism. If this header is present in an Egress response, its value overrides any locally defined operation (span) name on the client span. ``